js/client-chat.js

```diff
@@ -1237,6 +1237,13 @@
 					this.$chat.append('<div class="notice">' + Tools.sanitizeHTML(row.slice(1).join('|')) + '</div>');
 					break;
 
+				case 'notify':
+					if (!Tools.prefs('mute') && Tools.prefs('notifvolume')) {
+						soundManager.getSoundById('notif').setVolume(Tools.prefs('notifvolume')).play();
+					}
+					this.notifyOnce(row[1], "\"" + row.slice(2).join('|') + "\"", 'highlight');
```

Comment:
Incidentally: maybe `/wall` should highlight. That _is_ what its name is supposed to do.